apps/dialer/style/index.css

```diff
@@ -4,25 +4,19 @@ body {
   height: 100%;
   margin: 0;
   overflow-x: hidden;
-  color: lightgrey;
+  color: rgb(0, 0, 0);
+  font-family: "Readex Pro", sans-serif;
+  background: #FFFFFF;
+  padding: 1em;
+  text-align: left;
+  display: flex;
+  flex-direction: column;
```

Comment:
please move these to the `body { ... }` block instead. There is no point having them both on `html` and `body`.